cfpb/consumerfinance.gov
# Update wagtailmedia to 0.9.0

The [wagtailmedia library](https://github.com/torchbox/wagtailmedia/) that we use for managing podcast audio has had a few releases since we first added it to cf.gov. We use almost none of the features in wagtailmedia, so updating to 0.9.0 doesn't change anything for us. The [release notes](https://github.com/torchbox/wagtailmedia/releases) do mention that 0.9.0 is the version with Wagtail 2.15 support, so we may as well be up to date (and it looks like 0.6.0 will break if we go past Wagtail 2.15, so we may as well be ahead of the curve).

---

## Changes

- Bump wagtailmedia to 0.9.0

## How to test this PR

1. Visit the media library at `/admin/media/`. Make sure you can still manage the existing audio files and upload new ones and that there aren't any noticeable changes from what's in production right now other than:
    - The duration field is now optional
    - The duration field is now a float instead of an integer
3. Visit any podcast episode page (all the episodes are linked to from `/consumer-tools/educator-tools/students/financial-intuition/`). Confirm that the audio player still appears on the page and that the audio plays. Note that, depending on the environment you're testing in, you might have to upload a new audio file in the media library for the episode you're testing.

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
